### PR TITLE
Cleared grammatical error

### PR DESCRIPTION
**Pull request by spran180**

### CODE REVIEW

**Comment by AThousandShips** [comment]
These are all wrong

File: README.md [hunk 1]
```diff
@@ -19,14 +19,14 @@ Web-based platforms and [consoles](https://docs.godotengine.org/en/latest/tutori
 ## Free, open source and community-driven
 
 Godot is completely free and open source under the very permissive [MIT license](https://godotengine.org/license).
-No strings attached, no royalties, nothing. The users' games are theirs, down
+No strings attached, no royalties, nothing. The users games are theirs, down
```
Comment (AThousandShips) [inline README.md:22]: This is the plural possessive, it's "the games belonging to users"

File: README.md [hunk 2]
```diff
@@ -19,14 +19,14 @@ Web-based platforms and [consoles](https://docs.godotengine.org/en/latest/tutori
...
 to the last line of engine code. Godot's development is fully independent and
 community-driven, empowering users to help shape their engine to match their
 expectations. It is supported by the [Godot Foundation](https://godot.foundation/)
 not-for-profit.
 
 Before being open sourced in [February 2014](https://github.com/godotengine/godot/commit/0b806ee0fc9097fa7bda7ac0109191c9c5e0a1ac),
-Godot had been developed by [Juan Linietsky](https://github.com/reduz) and
+Godot has been developed by [Juan Linietsky](https://github.com/reduz) and
```
Comment (AThousandShips) [inline README.md:29]: This is wrong, before 2014 it had been developed, has is not correct here

File: README.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ for compilation instructions for every supported platform.
 Godot is not only an engine but an ever-growing community of users and engine
 developers. The main community channels are listed [on the homepage](https://godotengine.org/community).
 
-The best way to get in touch with the core engine developers is to join the
+The best way to get in touch with the core engine developers is by joining the
```
Comment (AThousandShips) [inline README.md:53]: Again grammar is correct already, "best way is to join" "best way is by joining" is not correct English

**Review by AThousandShips — CHANGES_REQUESTED** [review]
Thank you for contributing, but these are all wrong

**Comment by YuriSizov** [comment]
Indeed, these don't appear to be errors in need of fixing. I'm going to close this PR, but feel free to comment to explain why you think the changes need to be made.

**Pull request closed without merging**